README.md

```diff
@@ -10,11 +10,17 @@ A second note: Arduino Nanos remaining: 3 (send me a track to claim one!)
 
 You must have `sox` installed. Checkout out `http://sox.sourceforge.net/` to install it. (If you have a mac you can run `brew install sox`!) This handles taking our audio data and turning it into sound! I apologize for the dependency, maybe in the future it can be removed...
```

Comment:
```suggestion
You must have [`sox`](http://sox.sourceforge.net) installed. This handles taking our audio data and turning it into sound! I apologize for the dependency, maybe in the future it can be removed…
```

What do you think about rewording this part since we're calling out Homebrew below now?